next-tavla/app/(admin)/boards/components/TableHeader/ColumnHeader.tsx

```diff
@@ -11,10 +10,13 @@ function ColumnHeader({ column }: { column: TBoardsColumn }) {
     }
 
     return (
-        <div key={column} className={classes.header}>
+        <div
+            key={column}
+            className="flex items-center gap-1 border-b-2 border-b-[var(--header-border-color)] mb-2"
```

Comment:
skulle vi ikke ha variabler i stedet for å skrive `var(--header...)` her? sånn som `success` osv i configen 💭

Reply:
god catch, har glemt å legge denne inn som variabel i config-filen og byttet 👍